walkthrough/step05.md

```diff
@@ -0,0 +1,190 @@
+# Step05 - Auth middleware
+
+Now that we have our user passwords secured using bcrypt, and our cookies encoded with JWT, we can safley say that we took a step forward into having a secured website.
+
+There are many more steps that we can take to secure our website properly(links would be available at the bottom), but for now this is a good basis.
+
+<img src="" alt="" styles="text-align:center;" />
+
+**insert security related image here**
```

Comment:
Can't find one

Reply:
i'll just put a row of security related icon